Modules/_pickle.c

```diff
@@ -890,14 +890,15 @@ _Pickler_WriteFrameHeader(PicklerObject *self, char *qdata, size_t frame_len)
 }
 
 static int
-_Pickler_CommitFrame(PicklerObject *self)
+_Pickler_CommitFrame(PicklerObject *self, size_t frame_extension)
```

Comment:
I assumed that this is not part of the public API so it's fine to change the prototype of this function without caring about backward compat. Let me know if this is not the case.

Reply:
All static functions are private.